### PR TITLE
created first page

### PR DESCRIPTION
**Pull request by tensorwarriormayank880**
student admin page

### CODE REVIEW

File: src/Components/LoginPage/Login.scss
Comment (hereisadi) [inline src/Components/LoginPage/Login.scss:1]: it should be login.module.scss

File: src/Components/shared/Navbar/Navbar.jsx
Comment (hereisadi) [inline src/Components/shared/Navbar/Navbar.jsx:1]: unnecessary changes

**Review by hereisadi — CHANGES_REQUESTED** [review]

**Comment by hereisadi** [comment]
take pull of `dev` @tensorwarriormayank880

**Comment by tensorwarriormayank880** [comment]
newChanges

File: package.json [hunk 1]
```diff
@@ -10,10 +10,13 @@
...
   },
   "dependencies": {
+    "lottie-react": "^2.4.0",
     "prettier": "^3.0.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.15.0"
+    "react-icons": "^4.11.0",
+    "react-router-dom": "^6.15.0",
+    "scss": "^0.2.4"
```
Comment (hereisadi) [inline package.json:19]: no need to install scss. sass has already been listed in dev devDependencies

File: src/Components/LoginPage/Comp.json
Comment (hereisadi) [inline src/Components/LoginPage/Comp.json:1]: put the lottie json in `public` folder.

File: src/Components/LoginPage/Authentication.scss
Comment (hereisadi) [inline src/Components/LoginPage/Authentication.scss:1]: fileName should be Authentication.module.scss.

File: src/Components/LoginPage/Authentication.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    background-color: #40BDB6;
+    
+
+    .main_div {
+        // padding: 0;
+        min-width: 260px;
+        width: 660px;
+        // height: 100vh;
+    
+
```
Comment (hereisadi) [inline src/Components/LoginPage/Authentication.scss:15]: Also use relative font units only

**Review by hereisadi — CHANGES_REQUESTED** [review]

**Comment by hereisadi** [comment]
ping @tensorwarriormayank880

**Review by hereisadi — APPROVED** [review]

**Pull request closed without merging**